contracts/deployments/mainnet/.migrations.json

```diff
@@ -52,4 +52,7 @@
   "058_oeth_vault_value_checker": 1683419766,
   "055_curve_amo": 1683966879,
   "056_harvest_crv_limit": 17287144
-}
+  // TODO: when execute on timelock is called set that block timestamp as deploy one
```

Comment:
This isn't going to break anything by having comments in a .json file?

Reply:
It does. Let's hold off merge until the proposal is live and this file is updated.